docs/topics/i18n/translation.txt

```diff
@@ -2110,9 +2110,13 @@ message file specific to the project you are composing. The choice is yours.
 All message file repositories are structured the same way. They are:
 
 * All paths listed in :setting:`LOCALE_PATHS` in your settings file are
```

Comment:
"All paths..." needs some qualification now? 

* tuples are searched `language`, `domain`.
* Paths have an implicit domain of `django`

Reply:
This is explained on the next paragraph.